pkg/users/users.js

```diff
@@ -153,15 +161,33 @@ function AccountsPage() {
     } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo}
-                            current_user={current_user_info?.name} user={path[0]} shells={shells} />
+                current_user={current_user_info?.name} user={path[0]} shells={shells} />
         );
     } else return null;
 }
 
-async function getLogins() {
+async function getLastlogPath() {
+    try {
+        await cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" });
```

Comment:
So, /var/run/utmp does not exist when lastlog2 is installed? This surprises me, tbh. We have the same check already in account-details.jsx so I guess it's ok...

I think we should check for (and watch) /var/log/lastlog instead of /var/run/utmp. /var/log/lastlog seems to be the database used by lastlog. But we can attempt this in a separate PR.

Reply:
as far as I can tell if you enable lastlog2 you no longer have `/var/run/utmp`. If a distro is able to enable both we can always change to accommodate them